Userland/Applications/FileManager/PropertiesWindow.cpp

```diff
@@ -191,6 +195,53 @@ ErrorOr<void> PropertiesWindow::create_general_tab(GUI::TabWidget& tab_widget, b
     return {};
 }
 
+ErrorOr<void> PropertiesWindow::create_file_type_specific_tabs(GUI::TabWidget& tab_widget)
+{
+    auto mapped_file_or_error = Core::MappedFile::map(m_path);
+    if (mapped_file_or_error.is_error()) {
+        warnln("{}: {}", m_path, mapped_file_or_error.release_error());
+        return {};
+    }
+    auto mapped_file = mapped_file_or_error.release_value();
+
+    auto file_name_guess = Core::guess_mime_type_based_on_filename(m_path);
+    auto mime_type = Core::guess_mime_type_based_on_sniffed_bytes(mapped_file->bytes()).value_or(file_name_guess);
+
+    if (mime_type.starts_with("audio/"sv))
+        return create_audio_tab(tab_widget, move(mapped_file));
+
+    return {};
+}
+
+ErrorOr<void> PropertiesWindow::create_audio_tab(GUI::TabWidget& tab_widget, NonnullRefPtr<Core::MappedFile> mapped_file)
+{
+    auto loader_or_error = Audio::Loader::create(mapped_file->bytes());
+    if (loader_or_error.is_error()) {
+        warnln("Failed to open '{}': {}", m_path, loader_or_error.release_error());
+        return {};
+    }
+    auto loader = loader_or_error.release_value();
+
+    auto tab = TRY(tab_widget.try_add_tab<GUI::Widget>("Audio"_short_string));
+    TRY(tab->load_from_gml(properties_window_audio_tab_gml));
+
+    tab->find_descendant_of_type_named<GUI::Label>("audio_type")->set_text(TRY(String::from_deprecated_string(loader->format_name())));
+    auto duration_seconds = loader->total_samples() / loader->sample_rate();
+    tab->find_descendant_of_type_named<GUI::Label>("audio_duration")->set_text(TRY(String::formatted("{:02}:{:02}:{:02}", duration_seconds / 3600, duration_seconds / 60, duration_seconds % 60)));
```

Comment:
Note (to self): This display, like all runtime displays, break when the duration is 0 or very large, which can happen on files that don't specify their runtime, which is legal in FLAC (even in the streamable subset).